src/libs/SidebarUtils.ts

```diff
@@ -230,6 +230,11 @@ function getOptionData({
     const participantPersonalDetailList = Object.values(OptionsListUtils.getPersonalDetailsForAccountIDs(participantAccountIDs, personalDetails)) as PersonalDetails[];
     const personalDetail = participantPersonalDetailList[0] ?? {};
     const hasErrors = Object.keys(result.allReportErrors ?? {}).length !== 0;
+    let shouldShowViolations = false;
+    if (hasViolations && parentReportAction?.actionName === CONST.REPORT.ACTIONS.TYPE.IOU) {
+        const {IOUReportID} = parentReportAction?.originalMessage ?? {};
+        shouldShowViolations = !ReportUtils.isSettled(IOUReportID);
```

Comment:
I am still seeing RBR after approve request

<img width="1162" alt="Screenshot 2024-03-08 at 7 23 01 PM" src="https://github.com/Expensify/App/assets/108292595/31274c7f-effb-4c19-8699-a341d8a7f5be">

Reply:
@cead22 should we show RBR when approved but not reimbursed?

`ReportUtils` has `isReportApproved` (statusNum = APPROVED) and `isSettled` (statusNum = REIMBURSED) methods.

Reply:
Yes
- On the LHN only to the user that made the request
- On the money request view, to all users that have access

Reply:
> On the LHN only to the user that made the request

🟢 

> On the money request view, to all users that have access

I am not seeing violation on payer side

Reply:
That's probably because currently OpenReport is only returning violations for the user that creates the money request, but I submitted a backend fix for this

Reply:
> I am not seeing violation on payer side

That's probably because RequestMoney doesn't return violations, and doesn't push them to the relevant users either. There's another issue that OpenReport only returns violations to the user that made the money request.
 
I submitted a backend fix for the second issue, and I'm working on a backend solution for the first as well